### ISSUE TITLE
Error on `.wait(ms)` when `waitTimeout` is not set

### ISSUE THREAD

**Issue by raypatterson** [segmentio/nightmare#463]
Error appears after a `.scrollTo()` if that helps.

```shell
  nightmare queueing action "wait" +2ms
  nightmare running +4ms
  nightmare:actions .wait() for 2000ms +0ms
Error: .wait() timed out after undefinedmsec
    at Nightmare.<anonymous> (file.js:XX:XX)
    at Timer.listOnTimeout (timers.js:92:15)
```

Took a look at where the error was coming from, tried adding a `waitTimeout` and **it went away**.

Running Nightmare 2.1.3 with Node 5.5.0.

**Comment by rosshinkley** [comment]
I believe this was fixed in #443, which was released in (I think) 2.1.4.  Can you upgrade to that version or better?

**Comment by raypatterson** [comment]
Yep, that was it, thanks @rosshinkley. Next time I'll make sure to upgrade before reporting anything.
